lib/instadeploy.py

```diff
@@ -184,16 +186,26 @@ def build():
     if response.status_code != requests.codes.ok:
         raise MxBuildFailure("MxBuild failure", response.status_code, response.json())
 
+    result = response.json()
+    if result['status'] == 'Success':
+        try:
+            sync_project_files()
+            logger.info("Syncing project files ...")
+        except:
+            logger.warning("Syncing project files failed: %s", traceback.format_exc())
+    else:
+        logger.warning("Not syncing project files. MxBuild result: %s", result)
+
+    return result
+
+def sync_project_files():
```

Comment:
I do not agree here. Having many small functions will make things less readable because you constantly have to scroll through the code to see what is going on.